# Finnish translations for sos package.
# Ville-Pekka Vainio <vpivaini@cs.helsinki.fi>, 2009, 2010.
!=Project-Id-Version\: sos\nReport-Msgid-Bugs-To\: \nPOT-Creation-Date\: 2010-11-15 15\:02+0000\nPO-Revision-Date\: 2010-03-28 01\:03+0200\nLast-Translator\: Ville-Pekka Vainio <vpivaini@cs.helsinki.fi>\nLanguage-Team\: Finnish <laatu@lokalisointi.org>\nLanguage\: fi\nMIME-Version\: 1.0\nContent-Type\: text/plain; charset\=UTF-8\nContent-Transfer-Encoding\: 8bit\nPlural-Forms\: nplurals\=2; plural\=(n \!\= 1);\nGenerated-By\: pygettext.py 1.5\n

#: ../sos/sosreport.py:350
Could\ not\ create\ temporary\ directory.=V\u00e4liaikaista hakemistoa ei voitu luoda.

#: ../sos/sosreport.py:413
#, python-format
sosreport\ (version\ %s)=sosreport (versio %s)

#: ../sos/sosreport.py:438
#, python-format
plugin\ %s\ does\ not\ validate,\ skipping=liit\u00e4nn\u00e4inen %s on virheellinen, ohitetaan

#: ../sos/sosreport.py:467
#, python-format
plugin\ %s\ does\ not\ install,\ skipping=liit\u00e4nn\u00e4inen %s ei asennu, ohitetaan

#: ../sos/sosreport.py:563
no\ valid\ plugins\ found=kelvollisia liit\u00e4nn\u00e4isi\u00e4 ei l\u00f6ytynyt

#: ../sos/sosreport.py:567
The\ following\ plugins\ are\ currently\ enabled\:=Seuraavat liit\u00e4nn\u00e4iset ovat t\u00e4ll\u00e4 hetkell\u00e4 k\u00e4yt\u00f6ss\u00e4\:

#: ../sos/sosreport.py:573
No\ plugin\ enabled.=Yht\u00e4\u00e4n liit\u00e4nn\u00e4ist\u00e4 ei ole k\u00e4yt\u00f6ss\u00e4.

#: ../sos/sosreport.py:577
The\ following\ plugins\ are\ currently\ disabled\:=Seuraavat liit\u00e4nn\u00e4iset ovat t\u00e4ll\u00e4 hetkell\u00e4 poissa k\u00e4yt\u00f6st\u00e4\:

#: ../sos/sosreport.py:585
The\ following\ plugin\ options\ are\ available\:=Seuraavat liit\u00e4nn\u00e4isen asetukset ovat k\u00e4ytett\u00e4viss\u00e4\:

#: ../sos/sosreport.py:606
No\ plugin\ options\ available.=Liit\u00e4nn\u00e4isell\u00e4 ei ole asetuksia.

#: ../sos/sosreport.py:614
sosreport\ requires\ root\ permissions\ to\ run.=sosreportin k\u00e4ytt\u00f6 vaatii root-oikeudet.

#: ../sos/sosreport.py:621
no\ valid\ plugins\ were\ enabled=yht\u00e4\u00e4n kelvollista liit\u00e4nn\u00e4ist\u00e4 ei ole otettu k\u00e4ytt\u00f6\u00f6n

#: ../sos/sosreport.py:624
#, python-format
This\ utility\ will\ collect\ some\ detailed\ \ information\ about\ the\nhardware\ and\ setup\ of\ your\ %(distroa)s\ system.\nThe\ information\ is\ collected\ and\ an\ archive\ is\ \ packaged\ under\n/tmp,\ which\ you\ can\ send\ to\ a\ support\ representative.\n%(distrob)s\ will\ use\ this\ information\ for\ diagnostic\ purposes\ ONLY\nand\ it\ will\ be\ considered\ confidential\ information.\n\nThis\ process\ may\ take\ a\ while\ to\ complete.\nNo\ changes\ will\ be\ made\ to\ your\ system.\n\n=T\u00e4m\u00e4 ohjelma ker\u00e4\u00e4 yksityiskohtaisia tietoja\n%(distroa)s -j\u00e4rjestelm\u00e4n laitteistosta ja asetuksista.\nTiedot ker\u00e4t\u00e4\u00e4n arkistoon, joka tallennetaan /tmp-hakemistoon.\nT\u00e4m\u00e4 arkisto voidaan l\u00e4hett\u00e4\u00e4 k\u00e4ytt\u00f6tukeen.\n%(distrob)s k\u00e4ytt\u00e4\u00e4 t\u00e4t\u00e4 tietoa vain vianetsint\u00e4tarkoituksiin ja\ntietoa k\u00e4sitell\u00e4\u00e4n luottamuksellisesti.\n\nT\u00e4m\u00e4 toiminto voi kest\u00e4\u00e4 jonkin aikaa.\nJ\u00e4rjestelm\u00e4\u00e4n ei tehd\u00e4 muutoksia.\n\n

#: ../sos/sosreport.py:639
Press\ ENTER\ to\ continue,\ or\ CTRL-C\ to\ quit.\n=Paina ENTER jatkaaksesi, CTRL-C lopettaaksesi.\n

#: ../sos/sosreport.py:665
One\ or\ more\ plugins\ have\ detected\ a\ problem\ in\ your\ configuration.=Yksi tai useampi liit\u00e4nn\u00e4inen on huomannut ongelman j\u00e4rjestelm\u00e4ss\u00e4.

#: ../sos/sosreport.py:667
Please\ review\ the\ following\ messages\:=Tarkista seuraavat viestit\:

#: ../sos/sosreport.py:683
Are\ you\ sure\ you\ would\ like\ to\ continue\ (y/n)\ ?\ =Haluatko jatkaa (k/e)?

#: ../sos/sosreport.py:685
y=k

#: ../sos/sosreport.py:685
Y=K

#: ../sos/sosreport.py:688
n=e

#: ../sos/sosreport.py:688
N=E

#: ../sos/sosreport.py:713
\ \ Running\ plugins.\ Please\ wait\ ...=\  Suoritetaan liit\u00e4nn\u00e4isi\u00e4. Odota...

#: ../sos/sosreport.py:827
#, python-format
\ \ sosreport\ build\ tree\ is\ located\ at\ \:\ %s=\  sosreport-k\u00e4\u00e4nn\u00f6spuu on sijainnissa\: %s

#: ../sos/policyredhat.py:211
#, python-format
Please\ enter\ your\ first\ initial\ and\ last\ name\ [%s]\:\ =Sy\u00f6t\u00e4 etunimesi ensimm\u00e4inen kirjain ja sukunimesi [%s]\:

#: ../sos/policyredhat.py:214
Please\ enter\ the\ case\ number\ that\ you\ are\ generating\ this\ report\ for\:\ =Sy\u00f6t\u00e4 tapausnumero, jota varten olet tekem\u00e4ss\u00e4 t\u00e4t\u00e4 raporttia\:

#: ../sos/policyredhat.py:254
Creating\ compressed\ archive...=Luodaan pakattua arkistoa...

#: ../sos/policyredhat.py:284
Encrypting\ archive...=Salataan arkistoa...

#: ../sos/policyredhat.py:304
There\ was\ a\ problem\ encrypting\ your\ report.=Raportin salaamisessa oli ongelma.

#: ../sos/policyredhat.py:328
#, python-format
Your\ sosreport\ has\ been\ generated\ and\ saved\ in\:\n\ \ %s=Sosreport-raportti on luotu ja tallennettu sijaintiin\:\n  %s

#: ../sos/policyredhat.py:331
The\ md5sum\ is\:\ =Md5sum-tarkistussumma on\:

#: ../sos/policyredhat.py:333
Please\ send\ this\ file\ to\ your\ support\ representative.=L\u00e4het\u00e4 t\u00e4m\u00e4 tiedosto k\u00e4ytt\u00f6tukeen.

#: ../sos/policyredhat.py:355
No\ URL\ defined\ in\ config\ file.=Asetustiedostossa ei ole m\u00e4\u00e4ritelty osoitetta.

#: ../sos/policyredhat.py:362
Cannot\ upload\ to\ specified\ URL.=Annettuun osoitteeseen ei voida l\u00e4hett\u00e4\u00e4.

#: ../sos/policyredhat.py:399
There\ was\ a\ problem\ uploading\ your\ report\ to\ Red\ Hat\ support.=Raportin l\u00e4hett\u00e4misess\u00e4 Red Hatin k\u00e4ytt\u00f6tukeen oli ongelmia.

#: ../sos/policyredhat.py:401
#, python-format
Your\ report\ was\ successfully\ uploaded\ to\ %s\ with\ name\:=Raportin l\u00e4hetys kohteeseen %s onnistui, k\u00e4ytettiin nime\u00e4\:

#: ../sos/policyredhat.py:404
Please\ communicate\ this\ name\ to\ your\ support\ representative.=Kerro t\u00e4m\u00e4 nimi k\u00e4ytt\u00f6tukeen.
